quickwit/quickwit-indexing/src/actors/doc_processor.rs

```diff
@@ -413,12 +416,15 @@ impl DocProcessor {
                     processed_docs.push(processed_doc);
                 }
                 Err(error) => {
-                    warn!(
-                        index_id = self.counters.index_id,
-                        source_id = self.counters.source_id,
-                        "{}",
-                        error
-                    );
+                    let error_str = error.to_string();
```

Comment:
keying the hashmap on the string representation of the error is not a great idea. What if the error display name contains something dynamic? (like a docid or something)

Reply:
Then we would still have the 20 logs limit per minute